automation/project/Dependencies.scala

```diff
@@ -4,8 +4,8 @@ object Dependencies {
   val scalaV = "2.12"
 
   val jacksonV = "2.8.4"
-  val akkaV = "2.5.7"
-  val akkaHttpV = "10.0.10"
+  val akkaV = "2.6.1"
+  val akkaHttpV = "10.1.11"
```

Comment:
Update automation dependencies, partially to fix an error that comes up when running the tests locally:

```
An exception or error caused a run to abort: spray.json.package$.enrichString(Ljava/lang/String;)Lspray/json/RichString;
java.lang.NoSuchMethodError: spray.json.package$.enrichString(Ljava/lang/String;)Lspray/json/RichString;
	at org.broadinstitute.dsde.workbench.config.CommonConfig$CommonUsers.$init$(CommonConfig.scala:14)
	```